### ISSUE TITLE
support  multiple Bluetooth connect？

### ISSUE THREAD

**Issue by stylsw** [NordicSemiconductor/Android-BLE-Library#487]
Does it support connecting multiple Bluetooth peripherals?

**Comment by philips77** [comment]
Yes. For each peripheral you need a separate instance of `BleManager`. Each instance controls a different device.
Phones support up to 6 or 8 simultaneous connections, this depends on the phone.

**Comment by stylsw** [comment]
thank you!
